src/Components/Patient/DailyRounds.tsx

```diff
@@ -294,9 +294,14 @@ export const DailyRounds = (props: any) => {
         setIsLoading(false);
 
         if (obj) {
+          const roundsType = obj.rounds_type
+            ? obj.rounds_type.charAt(0).toUpperCase() +
+              obj.rounds_type.slice(1).toLowerCase() +
```

Comment:
Not a very readable code. It'd be cleaner if done something like:

```jsx
Notification.Sucess({
  msg: `Updated ${obj.rounds_type === "CRITICAL" ? "Critical" : "Normal"} log update succesfully.`
})
```

Reply:
![image](https://github.com/coronasafe/care_fe/assets/91010142/09c247a5-efdd-4be2-946f-04474f024cd6)
here its saying this comparison is unintentional and critical is not even exist there, there's ventilator type instead of critical, also there are other round types available like telemedicine and automated, I can't use this directly , otherwise if automated type would be created then the relevent notification wouldn't be shown there

Reply:
Then you can use text transforms.

Maybe: https://tailwindcss.com/docs/text-transform#transforming-text
Or some utility function from loadash.

Reply:
done